signature/signer.go

```diff
@@ -122,9 +122,10 @@ func (s *localSigner) PrivateKey() crypto.PrivateKey {
 	return s.key
 }
 
-// VerifyAuthenticity verifies the certificate chain in the given SignerInfo
-// with one of the trusted certificates and returns a certificate that matches
-// with one of the certificates in the SignerInfo.
+// VerifyAuthenticity iterates the certificate chain in signerInfo, for each
```

Comment:
nit: to link the type
```suggestion
// VerifyAuthenticity iterates the certificate chain in [signerInfo], for each
```